### DISCUSSION TITLE
Feature request: Hide all AI related prompts

### DISCUSSION THREAD

**Discussion by Lexicality** [mdn/mdn-community#451] [Platform] [answered]
![image](https://github.com/mdn/mdn-community/assets/847422/f815ef65-255e-4c1a-9bd5-9282b9d66cb8)
I do not wish to interact with your "AI Help" and would like to get rid of the constant prompts to do so.
Please either add a toggle to remove these prompts, or add a class (`ai-help` perhaps) to the elements so I can have my browser hide them for me

**Comment by Drbilal12** [comment]
> ![image](https://github.com/mdn/mdn-community/assets/847422/f815ef65-255e-4c1a-9bd5-9282b9d66cb8)
> I do not wish to interact with your "AI Help" and would like to get rid of the constant prompts to do so.
> Please either add a toggle to remove these prompts, or add a class (`ai-help` perhaps) to the elements so I can have my browser hide them for me

**Comment by bsmth** [comment, marked as answer]
The UI has been through some iterations since this Discussion was created, the items your referring to are no longer there, so I'm closing for now. Thank you :)
